loader/httploader/httploader.go

```diff
@@ -179,3 +186,35 @@ func (h *HTTPLoader) checkRedirect(r *http.Request, via []*http.Request) error {
 	}
 	return nil
 }
+
+type SafeDialer struct {
+	BlockNetworks  []*net.IPNet
+	BlockLoopback  bool
+	BlockPrivate   bool
+	BlockLinkLocal bool
+}
+
+func (s *SafeDialer) dialControl(network string, address string, conn syscall.RawConn) error {
```

Comment:
is it necessary to create a new struct `SafeDialer`? Seems this can also be wrapped under `HTTPLoader`?

Reply:
Yes, I thought it would maybe be cleaner to have it in it's own struct but it is not really necessary. I'll make that change.

Reply:
I believe the `BlockLoopback`, `BlockLinkLocal`, `BlockPrivate` will be exposed as options & cli config?

It would be easier to fit all parameters inside `HTTPLoader` then.